### DISCUSSION TITLE
Enter distrobox container default to specified home

### DISCUSSION THREAD

**Discussion by angelokimhui** [89luca89/distrobox#1178] [Q&A] [answered]
I don't want it to default in my host's home directory. Is there any way to change that behavior?

**Comment by 89luca89** [comment, marked as answer]
yes, use `-nw` and it will enter the custom home by default

**Reply by angelokimhui** [reply]
Thanks a lot!

**Reply by realyukii** [reply]
`--no-workdir/-nw:       always start the container from container's home directory`

but how to set container's home directory if the distro is already created? do I need to re-create it as described in [readme](https://github.com/89luca89/distrobox/blob/main/docs/useful_tips.md#create-a-distrobox-with-a-custom-home-directory)?
